components/admin_console/billing/company_info_edit.tsx

```diff
@@ -71,11 +67,15 @@ const CompanyInfoEdit: React.FC<Props> = () => {
 
     useEffect(() => {
         dispatch(getCloudCustomer());
-    }, []);
+    }, [dispatch]);
```

Comment:
I checked on the redux documenttion not long ago, and yes, dispatch has this problem, but it is supposed to be stable, unless you change the whole store (which you shouldn't). So... we will have to live with it :sweat_smile:

Reply:
Oh that makes sense that it would change if you swap the store. That makes sense as a case that can happen then even if it's a bit annoying that we can't override it still

Reply:
we could write a custom hook to "circumvent" that boilerplate code as well. Something like:

```
const useCustomDispatch = (action, dependencies) => {
    useEffect(() => dispatch(action()), [dispatch, ...dependencies]);
}
```

I am not sure (i did not do anything like that before) if this works this way though. Considering the nature of hooks this might need to be adjusted to a different structure.